### ISSUE TITLE
Internationalisation : Traduction

### ISSUE THREAD

**Issue by joshwhedon** [way-of-elendil/website#197]
Si on veut s'ouvrir a l'international, il faut supporter la traduction de contenu.
La premiere etape est probablement de simplement utiliser la traduction google pour le site.

**Comment by Babyface-187** [comment]
Il faudra tout de même tout relire parce que google.translate… J'ai pas trop confiance

**Comment by joshwhedon** [comment]
Oui, mais si le choix est entre pas de trad du tout, et google translate :D . Parce que je doute que qui que ce soit sera motive a tout traduire.

**Comment by Endalaust** [comment]
C'est pas toi le mec qui va traduire le fr vers l'anglais ?
Mouhahahaha

**Comment by joshwhedon** [comment]
C'est bien parce que y'a de bonnes chances que ca devra etre moi que je veux reduire au maximum.
